api/desecapi/urls/version_1.py

```diff
@@ -0,0 +1,61 @@
+from django.conf.urls import include, url
+from djoser.views import UserView
+from rest_framework.routers import SimpleRouter
+
+from desecapi import views
+
+
+tokens_router = SimpleRouter()
+tokens_router.register(r'', views.TokenViewSet, base_name='token')
+
+auth_urls = [
+    # Old user management
+    # TODO deprecated, remove
+    url(r'^users/create/$', views.UserCreateView.as_view(), name='user-create'),  # deprecated
+    url(r'^token/create/$', views.TokenCreateView.as_view(), name='token-create'),  # deprecated
+    url(r'^token/destroy/$', views.TokenDestroyView.as_view(), name='token-destroy'),  # deprecated
+
+    # New user management
+    url(r'^users/$', views.UserCreateView.as_view(), name='register'),
+
+    # Token management
+    url(r'^token/login/$', views.TokenCreateView.as_view(), name='login'),
+    url(r'^token/logout/$', views.TokenDestroyView.as_view(), name='logout'),
+    url(r'^', include('djoser.urls.authtoken')),  # note: this is partially overwritten by the two lines above
```

Comment:
This line has now moved up a bit, in comparison to the previous arrangement (https://github.com/desec-io/desec-stack/blob/224eeb9fef477332dc5eb3e67702d5fc6707cb37/api/api/urls.py#L18). Does this have any implications, and if so, what's the reasoning behind the change?

Reply:
It's rearranged and documented with line comments

Reply:
Yes, of course. Does it have any implications though (for example, has the way that routes can shadow each other changed)?

Reply:
imho it's functionally equivalent